sdp-utils.c

```diff
@@ -1371,7 +1374,9 @@ janus_sdp *janus_sdp_generate_answer(janus_sdp *offer, ...) {
 				}
 			} else {
 				/* Add rtpmap attribute */
-				janus_sdp_attribute *a = janus_sdp_attribute_create("rtpmap", "%d %s", pt, janus_sdp_get_codec_rtpmap(codec));
+				const char *codec_rtpmap = janus_sdp_get_codec_rtpmap(codec);
+				if(!codec_rtpmap) continue;
```

Comment:
Can you split this on two lines, per our code style?

Reply:
As a side note, why is there a `continue` for video, while for audio just a conditional? The `continue` as it is looks dangerous too, since this is in a `while` and you're not updating the pointer.

Reply:
> As a side note, why is there a `continue` for video, while for audio just a conditional? The `continue` as it is looks dangerous too, since this is in a `while` and you're not updating the pointer.

Yeah the continue is bad, re-pushing.

Reply:
Thanks for the quick fix! I'll need to make a few tests before I can merge, specifically in cases where no common codec is found. We recently fixed some issues in that scenario, and I want to be sure there won't be a regression. I'll do that when I get back to the office tomorrow.